README.md

```diff
@@ -424,6 +424,93 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+### Local Key Helper on Android
+On android platform a Local Key Helper is a protocol interface (`ILocalKeyHelper.java`), and its implementation (`LocalKeyHelperImpl.java`) is yet to be determined.
+
+Local KeyHelpers on the Android platform are implemented using [content providers](https://developer.android.com/guide/topics/providers/content-providers). These content providers allow browser applications, such as Chrome, to interact with them by querying and accessing specific APIs defined by the Local KeyHelpers.
+
+#### Content Provider Authority
+Each Local KeyHelper defines a content provider with a authority such as `com.contoso.localkeyhelper`. Different APIs are implemented as separate paths on this content provider.
+
+#### Browser Interaction with Local KeyHelpers
+To interact with Local KeyHelpers, browser applications must define a `<provider>` inside the `<queries>` tag in their manifest file with the same authority `com.contoso.localkeyhelper`. This allows the browser to query the Local KeyHelpers.
```

Comment:
To clarify, is this manifest within Android application? So, as an example, Authenticator will declare "com.microsoft.EntraLocalkeyHelper" and Chrome will be able to find it?

Reply:
But Chrome wouldn't be able to declare each possible local key helper. Is there any way to discover this dynamically without declaring each one?

Reply:
Updated to add registration of localkeyhelper with a browser application via Browser's ContentProvider.